new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java

```diff
@@ -193,23 +404,14 @@ public void testCreateMiiDomainWithMultiModeCM() {
       checkServiceExists(managedServerPrefix + i, domainNamespace);
     }
 
-    logger.info("Check the MaxCapacity setting of DataSource {0}", dsName);
-    String maxCapacityValue = getDSMaxCapacity(adminServerPodName, domainNamespace, dsName);
-    assertEquals("40", maxCapacityValue, 
-        String.format("Domain %s in namespace %s DataSource %s MaxCapacity is %s, instead of %s",
-            domainUid, domainNamespace, dsName, maxCapacityValue, "40"));
-
-    logger.info(String.format("Domain %s in namespace %s DataSource %s MaxCapacity is %s, as expected",
-            domainUid, domainNamespace, dsName, "40"));
-
-    logger.info("Check the MaxCapacity setting of DataSource {0}", dsName3);
-    maxCapacityValue = getDSMaxCapacity(adminServerPodName, domainNamespace, dsName3);
-    assertEquals("5", maxCapacityValue, 
-        String.format("Domain %s in namespace %s DataSource %s MaxCapacity is %s, instead of %s",
-            domainUid, domainNamespace, dsName3, maxCapacityValue, "5"));
+  }
 
-    logger.info(String.format("Domain %s in namespace %s DataSource %s MaxCapacity is %s, as expected",
-            domainUid, domainNamespace, dsName3, "5"));
+  @AfterAll
+  public void tearDownAll() {
+    // delete the domain image created in the test class
+    if (miiImageMultiModel != null) {
+      deleteImage(miiImageMultiModel);
```

Comment:
will this work if the domains are running?

Reply:
Probably not. Let me check. If not, I'll delete the domain first.

Reply:
Added delete domains before delete the image.

Reply:
I see deleting image works before deleting the domain..

Reply:
not sure if domain deletion will have issues though

Reply:
With the change, we are doing what we do in ItMiiDomain.

Reply:
In addition, I tested the current version and found no problem and the image was removed.

Reply:
I re-tested both cases, and either way worked without problem:).

Reply:
I recall that a Docker image cannot be deleted if there are containers using it. The containers have to be stopped first. So I think it is probably safer to delete the domain before deleting the image.